homework/insensitive-palindrom/palindromecheck.cpp

```diff
@@ -0,0 +1,16 @@
+#include "palindromecheck.hpp"
+
+bool is_palindrome(std::string Text)
+{
+    std::transform(Text.begin(), Text.end(), Text.begin(), [](auto character)
+
+                   { return std::toupper(character); });
+    Text.erase(std::remove_if(Text.begin(), Text.end(), [](auto character)
+                              { return character < 'A' || character > 'Z'; }),
+               Text.end());
+
+    std::string ReversedText;
+    ReversedText.resize(Text.size());
+    std::reverse_copy(Text.begin(), Text.end(), ReversedText.begin());
```

Comment:
not efficient. Better compare string starting form the begining and end. For instnace using std::equal. Also you need to chekc only half of stirng